public/css/showoff.css

```diff
@@ -72,6 +72,15 @@
   }
 }
 
+.content {
+  position: relative;
+  /* for some reason, setting top to 50% is pushing the content too far down, no idea why */
+  top: 42%;
```

Comment:
Can you make it top aligned for browsers that don't support transforms for graceful degradation?